Feature/documentation markdown (#111)
SyndemicsLab/hep-ce: docs/src/data.md

# Managing the Data

The HEPCE model is packaged as a library, but for ease of use we do also provide a basic command line interface executable for users to run the model. To this end, we make use of command line arguments and required input structures.

## Required Inputs

> There are two required input files that contain all the data we need. The first file is a configuration file titled `sim.conf`. The second is an SQLite Database file which contains multiple tables.

Reviewer: ```suggestion
There are two required input files that contain all necessary data:
- `sim.conf`
- `inputs.db`
```

### sim.conf

> The `sim.conf` file forms the backbone of the HEP-CE model. It governs the general event flow and the basic single-value parameters in the model. This file is meant to be changed by the users to determine what events occur to a person in the model and it does not contain tabular data. The general structure follows a prototypical `.ini` file with a header describing a set of key-value pairs. For HEPCE, there are 14 unique sets:

Reviewer: ```suggestion
The `sim.conf` file forms the backbone of the HEP-CE model. It governs the general event flow and the basic single-value parameters in the model. This file is meant to be changed by users to specify what events a person "experiences" in the model, and **it does not contain tabular data**. 

The file structure is the typical `.ini` structure--sections containing key-value pairs. There are 14 sections:
```

```ini
event = Aging, Pregnancy BehaviorChanges, FibrosisProgression, HCVInfection, HCVScreening, HCVLinking, VoluntaryRelinking, FibrosisStaging, HCVTreatment, Death
```

#### mortality

The mortality section governs the details of non-stratified death rates due to various conditions. An example is given below:

```ini
[mortality]
f4_infected         = 0.0106842628666393    # Monthly probability of dying due to F4 fibrosis while infected with HCV 
f4_uninfected       = 0.000641055772        # Monthly probability of dying due to F4 fibrosis while not infected with HCV
decomp_infected     = 0.028923195           # Monthly probability of dying due to Decompensated fibrosis while infected with HCV
decomp_uninfected   = 0.0017353917          # Monthly probability of dying due to Decompensated fibrosis while not infected with HCV
```

#### infection

> The infection section governs the details of non-stratified hepatitis-C infection parameters. An example is given below:

Reviewer: I don't think hepatitis-C is standard notation. If we've defined the shorthand HCV earlier in the docs, I think we'd be better off using that here. Otherwise, we should write out "hepatitis C virus" or "hepatitis C"
